plugins/slicer/MONAILabelReviewer/README.md

```diff
@@ -0,0 +1,75 @@
+# Purpose
```

Comment:
folder selection looks good to me.. which is MONAILabelReviewer under plugins/slicer..

Also we can update the Parent Readme about the two plugins `MONAILabel` and `MONAILabelReviewer` and CMAKE (for building the plugin can be shared for both cases)..

Reply:
Yes MONAILabelReviewer should be the 2nd module in the MONAILabel Slicer extension. https://github.com/Project-MONAI/MONAILabel/blob/1a4defa5090245bf0b5648d3c1f1f95c866039d1/plugins/slicer/CMakeLists.txt#L22